app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+    source_filename = parts[1]
+    destination_filename = parts[2]
```

Comment:
```suggestion
    cmd, source_filename , destination_filename = command.split()
```
This way would look better